src/util/strb.h

```diff
@@ -77,6 +77,15 @@ static inline int strb_error(strb *sb) {
   return sb->l == (size_t)-1;
 }
 
+/*
+ * Initialize at runtime an strb.
+ */
+
+static inline void strb_init(strb* sb){
+  const strb s = STRB_STATIC_INIT;
+  *sb = s;
```

Comment:
This can also be a `memset(b, 0, sizeof(strb))`.  We can assume that it is ok at this level.